metrics/src/metrics.rs

```diff
@@ -0,0 +1,212 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2021-Present Datadog, Inc.
+
+use ddcommon::tag::{parse_tags, Tag};
+use serde::Serialize;
+
+#[derive(PartialEq, Debug, Serialize, Clone)]
+pub struct Points {
+    pub timestamp: u64,
+    pub values: Vec<f64>,
+}
+
+impl Points {
+    pub fn new(values: Vec<f64>, timestamp: u64) -> Self {
+        Self { timestamp, values }
+    }
+}
+
+#[derive(PartialEq, Debug, Serialize, Clone)]
+pub enum MetricType {
+    Distribution,
+}
```

Comment:
There's a lot of different metrics types, but we only care about distribution metrics for now.

Reply:
Since the Point format will be different for distributions and Count/Gauges `(timestamp, Vec<point>)` `(timestamp, point)`
 
 I think the points should be a member of the enum, instead of having a `MetricType` enum
 
```rust
 #[derive(PartialEq, Debug, Serialize, Clone)]
pub enum MetricPoint {
    Distribution(Vec<f64>),

    // When you want to add other types
    Count(f64),
    Gauge(f64),
}
```